### ISSUE TITLE
magnetico 7.0 binary

### ISSUE THREAD

**Issue by pimboli** [boramalper/magnetico#175]
Hi,

can you please tell me if there is still a 7.0 binary available on the release page I only find source code.
If not can you pleas tell me how to compile source, I have no experience in go.

Thanks!

**Comment by Musickiller** [comment]
I have the same problem. A wiki or at least readme would be a great help to many people I think.

**Comment by FofeuLabs** [comment]
I recently submitted a pull request ( #178 ) that addresses this issue.
Just calling `make` should build both binaries as expected.
If you have any issues, you can ping me.

**Comment by Musickiller** [comment]
@FofeuLabs Any idea on how to make it work on Windows?)))

**Comment by FofeuLabs** [comment]
Not being a windows user, I don't know.
I see two ways:
1) Get cygwin (https://www.cygwin.com/), this will allow you to have a pseudo-unix environment. That's what I would do.
2) Go is supposed to work on windows. Look how to get go (https://golang.org/doc/install), go dep (https://github.com/golang/dep) and go-bindata (https://github.com/shuLhan/go-bindata). Then, you should be able to build binaries. Either by running make, or by copying the commands from the Makefile

**Comment by jangrewe** [comment]
I'm trying to build the Docker images (for which i've submitted PR #179 btw), but the `magneticow` build always fails:
```
Step 10/16 : RUN     make test-magneticow
 ---> Running in 4ba849bae727
go test github.com/boramalper/magnetico/cmd/magneticow/...
# github.com/boramalper/magnetico/cmd/magneticow
cmd/magneticow/handlers.go:143:15: undefined: Asset
cmd/magneticow/main.go:179:15: undefined: Asset
FAIL    github.com/boramalper/magnetico/cmd/magneticow [build failed]
make: *** [test-magneticow] Error 2
Makefile:29: recipe for target 'test-magneticow' failed
The command '/bin/sh -c make test-magneticow' returned a non-zero code: 2
```
I tried the fix from #178 , but that doesn't seem to work for me.

**Comment by FofeuLabs** [comment]
I admit, that I didn't consider the docker images in my PR
As far as I can tell, your PR is more complete than mine. Can you build the Docker images after applying your patches ?

**Comment by jangrewe** [comment]
Of course, i'm running them! ;-)

I probably should have mentioned here that i got them to work now, after doing some more changes.

**Comment by FofeuLabs** [comment]
I have a problem where I can't run both services at the same time
I get `FATAL   Could not open the database at `sqlite3:///path/to/database.sqlite3`: sql.DB.Ping: database is locked`
Did you encounter this problem ? If yes, how did you solve it ?

**Comment by jangrewe** [comment]
Hmm, i don't think the problem is running both at the same time, as SQLite does support concurrent access. Maybe you're running magnetico*d* twice?

**Comment by boramalper** [comment]
I'm sorry folks! I couldn't focus on the project because of university, but I now have a month so we'll make some progress. =)

I have just released [v0.7.0](https://github.com/boramalper/magnetico/releases/tag/v0.7.0) which includes binaries too. I'll check the merge requests in a second, and will update the documentation as well to include (better) installation instructions.

Besides, I desperately need some maintainers so if you are willing to contribute but don't know how, feel more than welcomed to ping me on Gitter or here and I'll help you familiarise with the project. =)

**Comment by boramalper** [comment]
Also, I haven't tested it on Windows, and honestly it's not a high priority for me nor an easy task as I currently don't have access to a Windows machine. 

@jangrewe **magneticow** build failed because the tests failed (as I've understood). Try again, it should work now. I also modified the travis configuration to use the Makefile supplied, so that whenever it passes on your local machine, it should also pass on travis, and vice versa. =)
